src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleDeclarations = sourceString.trim().split(';');
+  const styleObject = {};
+
+  for (const item of styleDeclarations) {
+    if (item.trim().length > 0) {
+      const resultItemSplit = item.split(':');
+      const key = resultItemSplit[0].trim();
+      const value = resultItemSplit[1].trim();
```

Comment:
You can improve your code like this
```suggestion
      const [key, value] = item.split(':').map(item => item.trim());
```